docs/resources/local_user.md

```diff
@@ -33,7 +33,7 @@ resource "windows_local_user" "this" {
 
 ### Optional
 
-- `account_expires` (String) Define when the local user account expires. If not specified, the user account never expires.<br>The string time format is the following: `2023-07-25T20:43:16Z` (see [Terraform timetypes](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework-timetypes@v0.3.0/timetypes#RFC3339)).
+- `account_expires` (String) Define when the local user account expires. If not specified, the user account never expires.<br>The string time format is the following: `2023-07-25T20:43:16Z` (ISO 8601 datetime/RFC3389, see [Terraform timetypes](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework-timetypes@v0.3.0/timetypes#RFC3339)).
```

Comment:
May not be neccessary since there are multiple examples

Reply:
For example: Linux "date" includes both "--iso-8601[=FMT]" and "--rfc-3339=FMT" as separate formatting options. These terraform timetypes examples only mention RFC3339, but I'm not sure if there actually would be any confusion.  I removed this suggestion from the pull request in the meantime, though.

Reply:
Any format that differs from the timetypes examples, will lead to an error. So the examples should be the way to go. Thanks!